openjdk/openjdk.mk

```diff
@@ -245,6 +245,15 @@ else ifneq (,$(findstring OpenJCEPlus, $(TEST_FLAG)))
 	FEATURE_PROBLEM_LIST_FILE:=-exclude:$(Q)$(JTREG_JDK_TEST_DIR)$(D)ProblemList-OpenJCEPlus.txt$(Q)
 endif
 
+# If we are on alpine, also use the exclude file specific to alpine.
+ifneq (,$(findstring alpine, $(SPEC)))
+	ifeq (,$(FEATURE_PROBLEM_LIST_FILE))
+		FEATURE_PROBLEM_LIST_FILE:=-exclude:$(Q)$(TEST_ROOT)$(D)openjdk$(D)excludes$(D)alpine$(D)ProblemList_openjdk$(JDK_VERSION)_alpine.txt$(Q)
+	else
+		FEATURE_PROBLEM_LIST_FILE+=-exclude:$(Q)$(TEST_ROOT)$(D)openjdk$(D)excludes$(D)alpine$(D)ProblemList_openjdk$(JDK_VERSION)_alpine.txt$(Q)
+	endif
+endif
+
```

Comment:
This needs one more check to deal with if ProblemList_openjdk$(JDK_VERSION)_alpine.txt doesn't exist. For example whenever the new version is available but the problemlist is not available yet or no need for that new version. 

[See example https://github.com/adoptium/aqa-tests/pull/6462/files#diff-c3d02f1e9a9f5dfed049a450b5d0fba4727b7b3d42f0a0771f2e41e408d6ef14L250](https://github.com/adoptium/aqa-tests/pull/6462/files#diff-c3d02f1e9a9f5dfed049a450b5d0fba4727b7b3d42f0a0771f2e41e408d6ef14L250-L254)

Reply:
I considered that, but opted not to so that if the file doesn't exist (or has the wrong name) we fail fast when we try to use it.

What do you think?

Reply:
The file should not be necessary especially if upstream support that later so I would prefer not to fail the job.

Reply:
Okie dokie. Will make that change. Is the rest ok?

Reply:
Fixed.

Test run with an incorrect ProblemList path (so the warning message can be tested):
https://ci.adoptium.net/job/Grinder/13887/

Test run with the correct ProblemList path: 
https://ci.adoptium.net/job/Grinder/13889/